docs/DeveloperGuide.md

```diff
@@ -856,6 +856,48 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `vdelete`, `vdelete x`, `...` (where x is larger than the list size or a non-integer)<br>
        Expected: Similar to previous.
 
+### Deleting an event
+
+1. Deleting an event.
+    1. Prerequisites: List all events using the `elist` command. Multiple events in the list. If the list is empty,
+       create it with the following example,
+       `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy m/100 packets b/100.00 vs/50`
+
+    1. Test case: `edelete 1`
+       Expected: The first event in the displayed event list is deleted successfully. Detail is shown in the status message.
+
+    1. Test case: `edelete`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
+
+    1. Test case: `edele`
+       Expected: No change to the event list panel. Invalid command format error is shown in the status message.<br>
```

Comment:
when 2 test cases produces the same expected result, combine them into one

1. Other incorrect delete commands to try: `efind`, `efind randomwordshere n/baking cookies`, `...` <br>
       Expected: `Invalid command format!` error message shown in the status message. Instructions on the correct format are shown as well.
       
like this

Reply:
or u can do it similar to what rx did and say `Expected: similar to previous`

Reply:
oh sounds good